tests/test_acasclient.py

```diff
@@ -3343,7 +3343,7 @@ def check_lot_exists_in_experiment(self, lot_corp_name, experiment_code_name):
         file_to_upload = get_basic_experiment_load_file(self.tempdir, project.names[PROJECT_NAME], restricted_lot_corp_name, file_name='4 parameter D-R.csv')
         response = self.client.\
             experiment_loader(file_to_upload, "bob", False)
-        self.assertTrue(check_lot_exists_in_experiment(self, restricted_lot_corp_name, response['results']['experimentCode']))
+        self.assertTrue(self.check_lot_exists_in_experiment(self, restricted_lot_corp_name, response['results']['experimentCode']))
```

Comment:
`self` shouldn't be passed now. It's the same for line 3362.

Reply:
Ah! Thanks.